tests/functional/sync_v2_reset_password.js

```diff
@@ -12,7 +12,7 @@ define([
   var config = intern.config;
 
   var PASSWORD = 'password';
-  var RESET_PASSWORD_URL = config.fxaContentRoot + 'reset_password?context=fx_desktop_v2&service=sync';
+  var RESET_PASSWORD_URL = config.fxaContentRoot + 'reset_password?context=fx_desktop_v2&service=sync&forceAboutAccounts=true&automatedBrowser=true';
```

Comment:
Didn't see any client docs on `forceAboutAccounts`. Might be helpful to describe what it does and the implications of having it there.

Reply:
> Didn't see any client docs on forceAboutAccounts

Adding it to the docs now!